COLE-readthis.md

```diff
@@ -0,0 +1,38 @@
+* Notes
+** I wasn't sure where to put my comments about steps/tasks/instructions in the code because everything is spread out over different files. If I put notes in one file you might not know whether I went there first or second or whatever. So here it is.
+** After reading the README 3 or 4 times I had no idea what I started to get a little upset because I wasn't sure what half of it meant, so I used the checklist that Ellen Nitchals sent out to guide me. Sorry. I know I'm supposed to follow the README but half the time they just confuse me more than I already was and I just go by the checklists as they make a lot more sense and are much easier to follow.
+** Question: single quotes or double quotes?
+
```

Comment:
Either or is fine in JS. If you don't have a manager or someone telling you that you have to use one, you can use either one. I generally use double quotes just in case I have a string like 

```
"Hi I have an apostrophe right about here ' and it doesn't affect the string"
```

Single quotes don't generally play nice with apostrophes.

P.S. Don't forget about the template strings (back ticks) which allow you to do string interpolation.